src/invidious/views/components/channel-information.ecr

```diff
@@ -0,0 +1,146 @@
+<% content_for "header" do %>
```

Comment:
Is this file in use somewhere? Doesn't seem to be rendered anywhere in this PR

Reply:
Probably leftovers of #2215 and/or #2546